src/validate.ts

```diff
@@ -15,12 +15,17 @@ const DISCONNECT_RESPONSE_SCHEMA = require('../intents/disconnect.response.schem
  * @param apiResponse User defined api response
  * @return Returns undefined if valid is true, returns an array of error(s) if valid is false.
  */
-function responseValidation(apiResponse: any, schema: any) {
+function responseValidation(apiResponse: object, schema: object) {
   const isValid = ajv.validate(schema, apiResponse);
-  if (isValid) {
-    return undefined;
+  try {
+    if (isValid) {
+      return undefined;
+    } else {
+      throw new Error();
```

Comment:
I don't understand why this would throw an error only to be caught in the same function. I believe there is some behavior here that should be re-evaluated to simplify the behavior.